spec/tiler_spec.coffee

```diff
@@ -1,92 +1,198 @@
-        expect(eventSpy).to.be.calledWith 'tile-2', 'tile-1'
+        it 'should set the reverse class', ->
+          expect($('#tile-is-reversible-1', $tiler)).to.not.have.class 'reverse'
+          expect($('#tile-is-reversible-2', $tiler)).to.not.have.class 'reverse'
 
-    context 'with an active class', ->
+    context 'with a custom animation', ->
       before ->
-        $('#go-to').tiler('goTo', 1, 'foo-animation')
+        clock = sinon.useFakeTimers()
+        $tiler = $('#custom-animation')
+        $tiler.tiler()
+        $tiler.tiler 'goTo', 2, 'foo-animation'
+        clock.tick 10
 
-      it 'should add the active class', ->
-        expect($('#go-to #tile-1').hasClass('foo-animation')).to.be.true
+      it 'should enable animation', ->
+        expect($tiler).to.not.have.class 'animation-disabled'
 
-    context 'when passing a boolean to active class', ->
+      it 'should add the animation class', ->
+        expect($('#tile-custom-animation-2', $tiler)).to.have.class 'foo-animation'
+        expect($('#tile-custom-animation-1', $tiler)).to.have.class 'foo-animation'
+
+    context 'when animation is true or not passed', ->
       before ->
-        $('#tile-1', '#go-to').data 'tiler-animation', 'goto-animate-1'
-        $('#tile-2', '#go-to').data 'tiler-animation', 'goto-animate-2'
+        clock = sinon.useFakeTimers()
+        $tiler = $('#animation-true-or-undefined')
+        $tiler.tiler()
+        $tiler.tiler 'goTo', 2, true
+        clock.tick 10
 
-      context 'when passing true', ->
-        before ->
-          $('#go-to').tiler('goTo', 2, true)
+      it 'should enable animation', ->
+        expect($tiler).to.not.have.class 'animation-disabled'
 
-        it 'should add the active class from the first tile', ->
-          expect($('#go-to #tile-2').hasClass('goto-animate-2')).to.be.true
+      it 'should use the animation of the tile', ->
+        expect($('#tile-animation-true-or-undefined-1', $tiler)).to.have.class 'tile-animation-true-or-undefined-animation-2'
+        expect($('#tile-animation-true-or-undefined-2', $tiler)).to.have.class 'tile-animation-true-or-undefined-animation-2'
 
-  describe 'refresh', ->
-    newWidth = null
-    newHeight = null
+    context 'when animation is false', ->
+      before ->
+        $tiler = $('#animation-false')
+        $tiler.tiler()
+        $tiler.tiler 'goTo', 2, false
+
+      it 'should not enable animation', ->
+        expect($tiler).to.have.class 'animation-disabled'
 
+  describe 'refresh', ->
     before ->
-      $('#refresh').tiler()
-      $('#refresh').tiler('goTo', 'tile-2')
-      $('#refresh').tiler('refresh')
+      $tiler = $('#refresh')
+      $tiler.tiler()
+      $tiler.tiler 'goTo', 2, false
+      $tiler.tiler 'refresh'
 
     it 'should stay on the current tile', ->
-      expect($('#refresh').attr('data-tiler-active-tile')).to.equal 'tile-2'
+      expect($tiler.attr('data-tiler-active-tile')).to.equal 'tile-refresh-2'
```

Comment:
is there anything else we can test to prove the method is firing? I would assume this is true without the refresh, right?